### PR TITLE
Update recipe.txt

### PR DESCRIPTION
**Pull request by janstieler**
Add Exception $error output to $alert.

### CODE REVIEW

File: content/1_docs/2_cookbook/4_forms/0_basic-contact-form/recipe.txt [hunk 1]
```diff
@@ -146,7 +146,7 @@ return function($kirby, $pages, $page) {
                 ]);
 
             } catch (Exception $error) {
-                $alert['error'] = "The form could not be sent";
+                $alert['error'] = "The form could not be sent:" . $error;
```
Comment (afbora) [inline content/1_docs/2_cookbook/4_forms/0_basic-contact-form/recipe.txt:149]:
```suggestion
                $alert['error'] = "The form could not be sent:" . $error->getMessage();
```

Comment (janstieler) [reply content/1_docs/2_cookbook/4_forms/0_basic-contact-form/recipe.txt:149]: Your right this is a nicer output.

**Comment by texnixe** [comment]
Not so sure if this is the kind of information you would want to show to a site visitor and therefore we only output a general error message.

**Comment by janstieler** [comment]
Hello Sonja,
okay that is a point so I would add a if-clause which is query the debug config:
```
 if(option('debug')):
    $alert['error'] = 'The form could not be sent: <strong>' . $error->getMessage() . '</strong>';
 else:
     $alert['error'] = 'The form could not be sent!';
endif;
```
The reason for this commit is that i had lost a couple of Time because I don't see that this example hat no $error-output.
Which is frustrating if you had copied this code .

Greetings
PS: Should I make a new commit with this or?

**Comment by texnixe** [comment]
Hey Jan, yes please!

**Pull request closed without merging**